text/????-separate-semantic-conventions.md

```diff
@@ -0,0 +1,115 @@
+# Separate Semantic Conventions
+
+Move Semantic Conventions outside of the main Specificaiton and version them
+separately.
+
+## Motivation
+
+We need to allow semantic conventions to evolve independently of the overall
+OpenTelemetry specification. Today, any breaking change in a semantic convention
+would require bumping the version number of the entirety of the OpenTelemetry
+specification.
+
+## Explanation
+
+A new github repository called `semantic-conventions` would be created in the
+OpenTelemetry organization.
+
+This would have the following structure:
+
+- Boilerplate files, e.g. `README.md`, `LICENSE`, `CODEOWNERS`, `CONTRIBUTING.md`
```

Comment:
Will this repository also include:
* [Definitions of stability for semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#semantic-conventions-stability)
* [Attribute requirement levels](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-requirement-level.md)

Reply:
+1

Reply:
Great call out.

SchemaURL is defined in the specification and interacts with key aspects of the specification, like Resource merge. I am not planning to "rip" that out.  Given the intertangled nature of SchemaURL, It's not clear to me yet HOW to tease those apart.

I'm adding this to open questions, as I think we can figure out the details as we do the merge.   My current thinking is that we need to check specification dependencies on these sections and we move everything that doesn't have a direct specification dependency.

Does this sound reasonable?